src/App.js

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from 'react';
 import Dropdown from './components/Dropdown';
-import USTopTenPositive from './components/USTopTenPositive';
-import USPositiveHistory from './components/USPositiveHistory';
+import USTopTenPositive from './components/unitedstates/USTopTenPositive';
+import USPositiveHistory from './components/unitedstates/USPositiveHistory';
```

Comment:
Are these components only ever going to be used for US?

Reply:
Seems like the components can be country agnostic. Any country specific details can be passed in as props?